src/store/ducks/opensrp/indexCasesDetails/index.ts

```diff
@@ -0,0 +1,154 @@
+import {
```

Comment:
I think the intention was to have the opensrp directory include only ducks that have entities that come from the OpenSRP API.  I would move this into a "data-warehouse" directory.  We may need to audit the opensrp ducks directory to see what we else we should move - could you create an issue for that please?

Reply:
Created #1358